index.js

```diff
@@ -67,6 +68,108 @@ function toggleColor(element) {
     }
 }
 
+// const toggleColor = document.querySelector("h1#header")
+
+// toggleColor.addEventListener('click', function(event) {
+//     if (event.target.style.color === "red") {
+//         event.target.style.color = "black"
+//     } else {
+//         event.target.style.color = "red"
+//     }
+// })
+
+// event.target will be whatever the user clicked on
+
+header.addEventListener('click', function() {
+    // console.log('clicked!')
+    toggleColor(header)
+})
```

Comment:
Nicely done! Using arrow functions (ES6 feature), you could also write this solution as shown below. This is the same exact functionality!
`header.addEventListener("click", () => toggleColor(header))`